person.rb

```diff
@@ -16,5 +17,10 @@ def of_age?
   def can_use_services?
     @parent_permision || of_age?
   end
+
+  def validate_name
+    corrector = Corrector.new
```

Comment:
- [ ] This should be an instance variable (`@corrector`) and it should be created in the **initialize** method. Kindly move this instantiation from here to the **initialize** method.